packages/reporter/src/hooks/hooks.tsx

```diff
@@ -42,11 +48,22 @@ export interface HooksProps {
   model: HooksModel
 }
 
-const Hooks = observer(({ model }: HooksProps) => (
-  <ul className='hooks-container'>
-    {_.map(model.hooks, (hook) => <Hook key={hook.id} model={hook} />)}
-  </ul>
-))
+const Hooks = observer(({ model }: HooksProps) => {
+  const hooksCount: { [key: string]: number } = {
```

Comment:
I think this logic should be moved into the test model. It could potentially be a computed property or maybe a method like `.hasMultipleHooks(_hookName_)`.